hazelcast-oss/get-hz-all-url.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ "${HZ_VERSION}" == *"SNAPSHOT"* ]]
+then
+    xml=$(curl -sSL https://oss.sonatype.org/content/repositories/snapshots/com/hazelcast/hazelcast-all/${HZ_VERSION}/maven-metadata.xml)
+    version=$(echo $xml | xpath -q  -e  '/metadata/versioning/snapshotVersions/snapshotVersion[1]/value' | grep -oP '(?<=<value>).*(?=</value>)')
```

Comment:
You always select the first value, are these snapshot versions always sorted and the first one is the latest one?

Reply:
`maven-metadata.xml` only holds the value for the latest snapshot version. The list consist of `jar`, `pom`, `sources` and `javadoc`. So it doesn't really matter which element of the list I get.